features/step_definitions/manage_user_account_spec.rb

```diff
@@ -0,0 +1,31 @@
+Given(/^a registered MAS user$/) do
+  step 'I register'
+end
+
+Given(/^that I am on the user account edit settings page in "(.*?)"$/) do |language|
+  visit "/#{language_to_locale(language)}/users/edit"
+
+  expect(page).to have_content(I18n.t('authentication.settings.title'))
+end
+
+When(/^I fill in the contact number$/) do
+  within('.l-registration') do
```

Comment:
Agreed. @Tr4pSt3R don't forget that after you fix, you rebase your new commits ok?